tests/testing_common.py

```diff
@@ -1056,6 +1083,29 @@ def _test_weighted_combination_of_adapters(self, model_id, config_cls, config_kw
             adapter_list[:2], weight_list[:2], "multi_adapter_linear_reweighting", combination_type="linear"
         )
 
+        # test ties re-weighting with multiple adapters
```

Comment:
Are we missing `combination_type="ties_svd"`?

Reply:
It's on line 1052